packages/phone-number-privacy/combiner/src/domain/endpoints/disable/action.ts

```diff
@@ -27,25 +34,4 @@ export class DomainDisableAction extends CombineAction<DisableDomainRequest> {
       session.logger
     )
   }
-
-  protected async receiveSuccess(
-    signerResponse: FetchResponse,
-    url: string,
-    session: Session<DisableDomainRequest>
-  ): Promise<DisableDomainResponse> {
-    const res = await super.receiveSuccess(signerResponse, url, session)
```

Comment:
why did we remove the call to super.receiveSuccess ?

Reply:
since we are now relying on the thresholdStateService to check if the domain is disabled, I didn't want to have the same logical check duplicated in `checkThresholdDisabled`. Since `DomainDisableAction` already extends `CombineAction` we just default to super.receiveSuccess anyways (just without the checkThresholdDisabled & fail early)